cloudomate/test/test_captchasolver.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import requests
+import json
+import os
+import base64
+import time
+
+from cloudomate.util.captchasolver import CaptchaSolver, ReCaptchaSolver
+from mock import patch, MagicMock
+
+
+class TestCaptchaSolver(unittest.TestCase):
+
+    def setUp(self):
+        self.captcha_solver = CaptchaSolver("213389asd8u912823")
+        self.recaptcha_solver = ReCaptchaSolver("213389asd8u912824")
+
+    @patch("json.loads")
+    @patch("requests.post")
+    def test_get_balance(self, mock_requests, mock_json):
+        self.captcha_solver.get_balance()  
```

Comment:
You're not testing anything here, simply invoking a method. Please add at least one `assert` statement for each test case.